src/Compilers/CSharp/Portable/Binder/Binder_Deconstruct.cs

```diff
@@ -660,7 +660,7 @@ private static string ExtractDeconstructResultElementName(BoundExpression expres
 
                 var deconstructMethod = ((BoundCall)result).Method;
                 var parameters = deconstructMethod.Parameters;
-                for (int i = (deconstructMethod.IsExtensionMethod ? 1 : 0); i < parameters.Length; i++)
+                for (int i = deconstructMethod.IsExtensionMethod ? 1 : 0; i < parameters.Length; i++)
```

Comment:
The parens here help keep the ?: from getting lost in the pool of punctuation for the for loop header.